README.md

```diff
@@ -1,18 +1,26 @@
 # BigImageViewer
 
-[ ![Download](https://api.bintray.com/packages/piasy/maven/BigImageViewer/images/download.svg) ](https://bintray.com/piasy/maven/BigImageViewer/_latestVersion)
+[![Download](https://api.bintray.com/packages/piasy/maven/BigImageViewer/images/download.svg)](https://bintray.com/piasy/maven/BigImageViewer/_latestVersion)
```

Comment:
Long lines make it [difficult](https://stackoverflow.com/questions/938105/readme-80-character-limit-or-just-let-the-users-editor-handle-it) to read for the user and so it I'm using the standard 80 character limit.